### ISSUE TITLE
Partial load/invalidation

### ISSUE THREAD

**Issue by Hugos68** [sveltejs/kit#11500]
### Describe the problem

Whenever having 2 data fetches in a load function and want to for example only invalidate the first you cannot do that, yeah you can use untrack but then you lose the automatic dependency tracking, you cannot currently manually invalidate 1 "part" of a load function.

### Describe the proposed solution

The solution would be simular to how ``hooks`` in sveltekit works where you can define multiple hooks, my proposal introduces multiple load functions and being able to "sequence" them, for example:
```js
// routes/products/+page.ts
async function loadProfile({ depends, fetch }) {
  depends('foo:profile');
  const res = await fetch(...);
  const profile = await res.json();
  return { profile };
};

async function loadProducts({ depends, fetch }) {
  depends('foo:products');
  const res = await fetch(...);
  const products= await res.json();
  return { products};
};

export async function load = multiple(loadProfile, loadProducts);
```
```html
<!-- routes/products/+page.svelte -->
<script>
  import { invalidate } from '$app/navigation';
  export let data;
</script>

<p>{profile.name}</p>

{#each data.products as product}
  <p>{product.name}</p>
{/each}

<button on:click={() => invalidate('foo:profile')}>Invalidate profile</button>
<button on:click={() => invalidate('foo:products')}>Invalidate products</button>
```

### Alternatives considered

You could also just export any function from the ``+page`` or ``+page.server`` with some sort of special prefix but that doesn't sound very nice, for example:
```js
export async function _loadProfile() { ... }; // This would count as a load function

export async function _loadProducts() { ... }; // This would count as a load function

export async function loadFooBar() { ... }; // This would NOT count as a load function because it doesn't have the ``_load`` prefix
```

### Importance

would make my life easier

### Additional Information

This is pretty crucial because each time you invalidate because you need to invalidate only a partial piece of the load function you are making more network requests to your backend, and although caching helps with this, you shouldn't *have* to be forced to rerun everything inside your load function when only a small part needs to be invalidated.

**Comment by dukesteen** [comment]
I like this multiple() syntax

**Comment by joetifa2003** [comment]
This will make it so much easier.
The only alternative I see is to use:enhance and do updates yourself via fetch calls for the specific parts that need to be updated from the load function.

**Comment by Izobov** [comment]
Alternative solution, not sure about this:
you can add another file +page.server.ts in your path folder

But it would be difficult to keep track of which one is for which purpose without naming them like
+products.server.ts and +profile.server.js.

**Comment by Hugos68** [comment]
@Izobov I don't think that makes much sense because all exports like form actions, prerender, ssr and csr are all part of a +page.server.ts

**Comment by MentalGear** [comment]
I support this proposal, but would make the function name clearer, e.g. `loadMultiple` or `loadComposite`.

**Comment by AlbertMarashi** [comment]
Suggested a great enhancement of this over at the following discussion:

- https://github.com/sveltejs/kit/discussions/13336

**Comment by jacobbogers** [comment]
@Hugos68  to add to this discussion,  how about

```javascript
 export const loaders = [ loader1, loader2] 
```


My 2cts

**Comment by filiabel** [comment]
I guess this is mostly solved with the new [remote functions](https://svelte.dev/docs/kit/remote-functions)?

**Comment by dummdidumm** [comment]
That is correct! Closing for this reason.

**Comment by Hugos68** [comment]
Awesome!
